docs/ex/get-started/02-http-server/main.go

```diff
@@ -29,28 +29,28 @@ import (
 	"go.uber.org/fx"
 )
 
-// region provide-server
+// --8<-- [start:provide-server]
 func main() {
-	// region app
+	// --8<-- [start:app]
 	fx.New(
 		fx.Provide(NewHTTPServer),
-		// endregion provide-server
+		// --8<-- [end:provide-server]
 		fx.Invoke(func(*http.Server) {}),
-	// region provide-server
+	// --8<-- [start:provide-server]
```

Comment:
Since these no longer get concatenated, we'll probably want to split `provide-server` into `provide-server-1` and `provide-server-2` here, right?

Reply:
Oops, yeah, you're right. I thought I got all cases, but I remember now that this was one of the first examples I moved over and was still testing out what did and didn't work. Fixing.

Reply:
Found another couple cases; fixing them all.